packages/playground/src/services.ts

```diff
@@ -13,6 +14,7 @@ import "./style.css";
 
 export function attachServices(host: BrowserHost) {
   monaco.languages.register({ id: "cadl" });
+  monaco.languages.setLanguageConfiguration("cadl", CadlLanguageConfiguration as any);
```

Comment:
Why is there a cast needed here?

Reply:
This is unfortunate we have to do that but without typescript is not happy that we are passing a `readonly` array for the tuple properties to a non readonly type but then if I remove `as const` then typescript doesn't get the full type of the `CadlLanguageConfiguraiton`  and it fails as well.
Options are: 
- keep it as it is
- import monaco editor in the compiler just for the type
- copy the type in the editor

Reply:
I see. I vote leave it with the cast. :)